docs/src/main/asciidoc/rest-client-guide.adoc

```diff
@@ -268,3 +269,7 @@ You should see a JSON object containing some basic information about Greece.
 
 As usual, the application can be packaged using `mvn clean package` and executed using the `-runner.jar` file.
 You can also generate the native executable with `mvn clean package -Pnative`.
+
+== Further reading
+
+link:https://download.eclipse.org/microprofile/microprofile-rest-client-1.2.1/microprofile-rest-client-1.2.1.html[MicroProfile Rest Client specification]
```

Comment:
Very minor nitpicking: even if there is only one element, I would use a list to have a better rendering.